onitu/api/__main__.py

```diff
@@ -30,16 +30,134 @@
 escalator = Escalator(session)
 logger = Logger("REST API")
 
+ONITU_READTHEDOCS = "https://onitu.readthedocs.org/en/latest/api.html"
+HOST = escalator.get(u'api:host') or 'localhost'
```

Comment:
Sometimes I go through the sources looking for code smells like that, and I don't really want to have to go through all the code to find out if there is always a value or not.
So yeah, if you could use `default=` that would be great :) .